src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/SettleClaimMarkPaidFullCallbackHandler.java

```diff
@@ -37,13 +56,62 @@ public List<CaseEvent> handledEvents() {
     }
 
     private CallbackResponse validateState(CallbackParams callbackParams) {
-        CaseData caseData = callbackParams.getCaseData();
+        var caseData = callbackParams.getCaseData();
+        final var caseDataBuilder = caseData.toBuilder();
         List<String> errors = new ArrayList<>();
 
         SettleClaimHelper.checkState(caseData, errors);
-
+        if (errors.isEmpty()) {
```

Comment:
Do we need this check?

Reply:
> Do we need this check for errors as we are initialising this as new ArrayList above?

An error might be added on the checkState call